lua/apisix/plugins/jwt-auth.lua

```diff
@@ -75,7 +82,7 @@ function _M.check_schema(conf)
     end
 
     if not conf.secret then
-        conf.secret = core.id.gen_uuid_v4()
+        conf.secret = ngx_encode_base64(resty_random.bytes(32, true))
```

Comment:
need more test cases, for example:
- no `secret` and no `base64_secret`
- has 'secret' and no `base64_secret`
etc.

Reply:
need doc for this PR too

Reply:
> need more test cases, for example:
> 
> * no `secret` and no `base64_secret`
> * has 'secret' and no `base64_secret`
>   etc.

don't get this, base64_secret is a boolean flag, it's not another secret.

Reply:
> need doc for this PR too

do you mean need to change jwt-auth doc?